### PR TITLE
Add github actions workflow

### PR DESCRIPTION
**Pull request by binoue**
This PR switches the etcdpasswd CI from CircleCI to GitHub Actions.

Signed-off-by: binoue <banji-inoue@cybozu.co.jp>

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          name: etcdpasswd
+          path: /tmp/upload
+      - run: |
+          TAG=${GITHUB_REF#refs/tags/v}
+          OWNER=$(echo ${{ github.repository }} | cut -d '/' -f 1)
+          REPOSITORY=$(echo ${{ github.repository }} | cut -d '/' -f 2)
+          if ${{ contains(github.ref, '-') }}; then
+            PRERELEASE="-prerelease"
+          fi
+          ghr -t ${{ secrets.GITHUB_TOKEN }} -u ${OWNER} -r ${REPOSITORY} -n ${TAG} ${PRERELEASE} -b "See [CHANGELOG.md](./CHANGELOG.md) for details." ${TAG} /tmp/upload
```
Comment (masa213f) [inline .github/workflows/main.yaml:117]: The URL of a release page is as follows.
https://github.com/cybozu-go/etcdpasswd/releases/tag/v1.1.0

So this link will open the following link, I think.
https://github.com/cybozu-go/etcdpasswd/releases/tag/v1.1.0/CHANGELOG.md
(or https://github.com/cybozu-go/etcdpasswd/releases/tag/CHANGELOG.md)

But the actual URL for CHANGELOG.md is below.
https://github.com/cybozu-go/etcdpasswd/blob/v1.1.0/CHANGELOG.md

**Review by masa213f — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -5,9 +5,12 @@ DOCDIR := debian/usr/share/doc/etcdpasswd
 CONTROL := debian/DEBIAN/control
 SUDO = sudo
 
+ETCD_VER=v3.3.10
```
Comment (yokaze) [inline Makefile:8]: I think most of neco products omit `v` from the software version.

**Review by yokaze — DISMISSED** [review]

File: .github/workflows/main.yaml (hunk 1 above)
Comment (binoue) [reply .github/workflows/main.yaml:117]: I think this is fine.
See [the releases in coil](https://github.com/cybozu-go/coil/releases)

**Review by masa213f — DISMISSED** [review]
LGTM

**Review by masa213f — APPROVED** [review]
LGTM

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+      - run: docker run -d --network host quay.io/coreos/etcd:v3.3
+      - run: make SUDO="sudo" setup
+      - run: make test
+      - name: Set version name
```
Comment (masa213f) [inline .github/workflows/main.yaml:24]:
```suggestion
      - name: Set version
```

**Review by masa213f — APPROVED** [review]

**Pull request merged**